yarn.lock

```diff
@@ -6128,7 +6128,6 @@ query-string@^4.1.0:
 
 quibble@^0.6.7, quibble@connorjclark/quibble#mod-cache:
   version "0.6.14"
-  uid "68d53b087d4c9117cc86c7e5f19e7953a219582b"
```

Comment:
This was an intentional update from your quibble stuff right?

Reply:
making any change to the deps results in this delta even on master, so best to just ignore i guess.